apps/cli/test/builder/docker.test.ts

```diff
@@ -0,0 +1,68 @@
+import fs from "fs-extra";
+import path from "path";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { build } from "../../src/builder/docker";
+import { DockerDriveConfig } from "../../src/config";
+
+describe("docker_builder", () => {
+    const image = "cartesi/sdk:0.11.0";
+
+    beforeAll(({ name }) => {
+        const destination = path.join(__dirname, "output", name);
+        fs.mkdirpSync(destination);
+    });
+
+    it("should not build withoua correct context", async ({ task }) => {
+        const destination = path.join(__dirname, "output", task.suite!.name);
+        const drive: DockerDriveConfig = {
+            builder: "docker",
+            context: ".",
+            dockerfile: "Dockerfile",
+            extraSize: 0,
+            format: "ext2",
+            tags: [],
+            image: undefined,
+            target: undefined,
+        };
+        await expect(build(task.id, drive, image, destination)).rejects.toThrow(
+            "exit code 1",
+        );
+    });
+
+    it("should not build a non-riscv image", async ({ task }) => {
+        const destination = path.join(__dirname, "output", task.suite!.name);
+        const drive: DockerDriveConfig = {
+            builder: "docker",
+            context: path.join(__dirname, "data"),
+            dockerfile: path.join(__dirname, "data", "Dockerfile.nonriscv"),
+            extraSize: 0,
+            format: "ext2",
+            tags: [],
+            image: undefined,
+            target: undefined,
+        };
+        await expect(build(task.id, drive, image, destination)).rejects.toThrow(
+            "Invalid image Architecture: arm64. Expected riscv64",
```

Comment:
I get the samething when running locally, since here it's `amd64`. Maybe we should get this from the host runnign the tests.

Reply:
reduced the error check to only "Expected riscv64"